unit_test/tstForce.hpp

```diff
@@ -1245,6 +1254,80 @@ TEST( TEST_CATEGORY, test_force_lps_multi )
     auto models = CabanaPD::createMultiForceModel(
         particles, CabanaPD::AverageTag{}, model1, model2 );
 
+    static_assert( std::is_same_v<decltype( model1 )::force_tag,
+                                  typename decltype( models )::force_tag> );
+    static_assert( std::is_same_v<typename model_type::model_tag,
+                                  typename decltype( models )::model_tag> );
+    static_assert( std::is_same_v<CabanaPD::Fracture,
+                                  typename decltype( models )::fracture_type> );
+    static_assert( std::is_same_v<CabanaPD::TemperatureIndependent,
+                                  typename decltype( models )::thermal_type> );
```

Comment:
These assertions definitely make sense with the current test set up, but just noting that the `force_tag` must match, `model_tag` does not, `fracture_type`, but `thermal_type` must either be temperature independent or not (dependent/heat transfer)